tripleawarclub.org/public_html/dice/MARTI.php

```diff
@@ -0,0 +1,68 @@
+<?php
+error_reporting(E_ALL);
+ini_set('display_errors', '1');
+?>
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>	M.A.R.T.I. Server --> "more accurate rolls than irony"- server</title>
+	</head>
+	<body>
+		<?php
+		error_reporting(E_ALL);
+		ini_set('display_errors', '1');
```

Comment:
Duplicate code, this is already execute before the doctype is sent...
I think removing the part before the doctype is probably better!